mapApp/templates/mapApp/incident_form.html

```diff
@@ -299,17 +299,17 @@ <h4 class="modal-title" id="incidentForm">{% trans "Submit a new report" %}</h4>
                 $(witness).html("I was not directly involved");
                 $("#div_id_impact select").append(witness);
                 $("#div_id_impact select").val('Witness').change();
-                $('#div_id_bike_lights > label').text("Was the rider using bike lights?");
-                $('#div_id_direction > label').text("What direction was the rider heading?");
-                $('#div_id_turning > label').text("How was the rider moving?");
-                $('#div_id_bicycle_type > label').text("What type of bicycle was involved in the incident?");
-                $('#div_id_helmet > label').text("Was the rider wearing a helmet?");
+                $('#div_id_bike_lights > label').text(gettext("Was the rider using bike lights?"));
+                $('#div_id_direction > label').text(gettext("What direction was the rider heading?"));
+                $('#div_id_turning > label').text(gettext("How was the rider moving?"));
+                $('#div_id_bicycle_type > label').text(gettext("What type of bicycle was involved in the incident?"));
+                $('#div_id_helmet > label').text(gettext("Was the rider wearing a helmet?"));
             } else {
                 $('#near-miss-details').find('#div_id_witness_vehicle').hide();
                 $('#collision-details').find('#div_id_witness_vehicle').hide();
                 $("#div_id_witness_vehicle select").val('NA');
-                $('#div_id_incident_with > label').text("What sort of object did you collide or nearly collide with?*");
-                $('#div_id_injury > label').text("Were you injured?*");
+                $('#div_id_incident_with > label').text(gettext("What sort of object did you collide or nearly collide with?*"));
+                $('#div_id_injury > label').text(gettext(gettext("Were you injured?*")));

```

Comment:
Is this double 'gettext' a copy/paste error?

Reply:
Good catch! I corrected the error and updated the pull request. Thanks!

Reply:
Thanks!